src/common/swagger/schema/upload.yml

```
@@ -0,0 +1,19 @@
+      description: Upload image success response
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              success:
+                type: boolean
+                example: true
+              message:
+                type: string
+                example: 'Image uploaded successfully'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)